src/main/webapp/modal-activator.js

```diff
@@ -35,4 +35,34 @@ function closeModal(modal) {
   if (modal == null) return;
   overlay.classList.remove('active');
   modal.classList.remove('active');
+}
+
```

Comment:
Adding a few comments to this code would be excellent!

Reply:
Addressed: 
6f64b29